modules/unavailability-of-the-kernel-driver.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/commonly-occurring-error-conditions.adoc
+:_module-type: CONCEPT
+[id="unavailability-of-the-kernel-driver_{context}"]
+= Unavailability of the kernel driver
+
+The Collector will determine if it has a kernel driver for the node's particular kernel version. The Collector makes an attempt to download a driver from the Sensor after first searching the local storage for one with the appropriate version and type. The following logs indicate that neither a local kernel driver nor a driver from the Sensor are present:
```

Comment:
```suggestion
Collector determines if it has a kernel driver for the node's kernel version. Then, Collector attempts to download a driver from the Sensor after first searching the local storage for one with the correct version and type. The following logs indicate that neither a local kernel driver nor a driver from the Sensor is present:
```

Reply:
Additionally, in `Then, Collector attempts to download a driver from the Sensor after first searching the local storage for one with the correct version and type.` Since Collector first searches local storage, it should come first in the sentence.